src/apps/wallet/tokens/token-skeleton.tsx

```diff
@@ -0,0 +1,22 @@
+import styles from './token-skeleton.module.scss';
+import { Skeleton } from '@zero-tech/zui/components';
+
+export const TokenSkeleton = () => {
+  return (
+    <div className={styles.tokenSkeleton}>
```

Comment:
@colbr - guys shall we decide on camelcase / capitalize for classes? very minor but worth aligning